codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/protocol/ServerProtocolTestGenerator.kt

```diff
@@ -842,7 +846,7 @@ class ServerProtocolTestGenerator(
     private fun assertOk(rustWriter: RustWriter, inner: Writable) {
         rustWriter.rust("#T(", RuntimeType.ProtocolTestHelper(codegenContext.runtimeConfig, "assert_ok"))
         inner(rustWriter)
-        rustWriter.rust(");")
+        rustWriter.write(");")
```

Comment:
Using `rustWriter.rust` here caused IntelliJ to report an error because `);` is not a valid rust snippet.